packages/workflow/src/records/handler/reject.ts

```diff
@@ -10,14 +10,13 @@
  */
 
 import * as z from 'zod'
-import { getLoggedInPractitionerResource } from '@workflow/features/user/utils'
 import { createRoute } from '@workflow/states'
 import { getToken } from '@workflow/utils/authUtils'
 import { validateRequest } from '@workflow/utils/index'
 import { toRejected } from '@workflow/records/state-transitions'
-import { sendBundleToHearth } from '@workflow/records/fhir'
 import { indexBundle } from '@workflow/records/search'
 import { auditEvent } from '@workflow/records/audit'
+import { isTaskBundleEntry } from '@workflow/../../commons/build/dist/types'
```

Comment:
odd import

Reply:
I need to change that, thanks! I wonder if there was some way to let tsserver know what to use for the imports from commons.

Reply:
I have run into it as well :/ VSCode Intellisense doesn't seem to pick up the auto imports, wonder if it's something we could configure better with tsconfig.